### DISCUSSION TITLE
Github Education request

### DISCUSSION THREAD

**Discussion by MSheekh6** [community/community#144393] [GitHub Education]
### Hi everyone,

I recently applied for the github education free benefits, and got approved however i have yet to recieve either an email about such benefits , or any access to the benefits. It has been roughly a week since I applied, is this normal?

**Comment by Rawknee-69** [comment]
@MSheekh6 well here is the answer to your question and if you find this helpful please mark this as an answer.

No, certainly not its not really normal for this to happen but you can contact the GitHub support team and ask them about your current status..

**Reply by MSheekh6** [reply]
okay will do, thanks

**Comment by davevad93** [comment]
Hi @MSheekh6 ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @MSheekh6 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by lllHunterlllSV** [comment]
**Hello! 👋**

Thanks for reaching out! It’s great to hear that you’ve been approved for the **GitHub Education Pack** 🎉. I completely understand your concern about not yet receiving the benefits or an email confirmation.

Here’s a breakdown of what could be happening:

- ⏳ **Processing Time**: After approval, it can sometimes take **a few days** for the benefits to be fully activated in your account. While it’s typically fast, some delays can happen depending on the system processing.
- 📧 **Email Confirmation**: Make sure to check your inbox (and spam/junk folder) for any confirmation emails from GitHub Education. Occasionally, these messages can be filtered by email clients.
- 🔑 **Accessing Benefits**: If you’ve been approved but haven’t seen the benefits yet, you should be able to access them directly via your [GitHub Education Dashboard](https://education.github.com/pack). The benefits should appear there once everything is processed.
- 🔄 **Waiting Period**: In some cases, it might take **a bit longer than a week**, especially during busy times like the beginning of the school year. However, if you still haven’t received the benefits or an email, it’s a good idea to follow up.

If you still don’t see any changes after another few days, I recommend reaching out to [GitHub Support](https://support.github.com/) to check the status of your account and benefits.

I hope this helps, and best of luck with your GitHub Student Pack! 🎉
